xc/xc7/boards.cmake

```diff
@@ -95,3 +88,32 @@ add_xc_board(
   PROG_CMD "${OPENOCD} -f board/digilent_nexys_video.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
 )
 
+add_xc_board(
+  BOARD zedboard-full
+  DEVICE xc7z020
+  PACKAGE test
+  PART xc7z020clg484-1
+)
+
+add_xc_board(
+  BOARD microzed-full
+  DEVICE xc7z020
+  PACKAGE test
+  PART xc7z020clg484-1
+)
+
+add_xc_board(
+  BOARD pynqz1-full
+  DEVICE xc7z020
+  PACKAGE test
+  PART xc7z020clg400-1
+  PROG_TOOL ${OPENOCD_TARGET}
+  PROG_CMD "${OPENOCD} -f ${PRJXRAY_DIR}/utils/openocd/board-digilent-pynqz1.cfg -c \\\"init $<SEMICOLON> pld load 0 \${OUT_BIN} $<SEMICOLON> exit\\\""
+)
+
+add_xc_board(
+  BOARD marszx3-full
+  DEVICE xc7z020
+  PACKAGE test
+  PART xc7z020clg484-1
```

Comment:
3 of these 4 boards have no PROG_CMD.  Is there not a supported OpenOCD config for these boards?  @mithro Should we have a place to track missing OpenOCD configuration support?

Reply:
Not sure if there are openocd configs for the other boards already, but if there are I can't test if they work as I only have a pynq z1.

Reply:
In general it is probably a good idea to leave off the untested configurations, or comment them out.  Making a small follow on PR with board support once it gets tested on other targets it preferable to checking in untested configurations.

We do have some testing via the vendor tool CI, but ideally we have both the "works on a board" and a green result from the CI.